modules/ob-defining-param-values.adoc

```diff
@@ -0,0 +1,23 @@
+// This module is included in the following assembly:
+//
+// builds/configuring-openshift-builds.adoc
+
+:_content-type: CONCEPT
+[id="ob-defining-param-values_{context}"]
+= Defining parameter values for a build
```

Comment:
I see you changed them to "Define". That's still a verb - usually concept would be more like a noun. I won't hold up the PR over it though since everything else looks good. Just something to consider for next time!